src/shipit_static_analysis/requirements.nix

```diff
@@ -91,6 +91,24 @@ let
 
 
 
+    "RBTools" = python.mkDerivation {
+      name = "RBTools-0.7.10";
+      src = pkgs.fetchurl { url = "https://pypi.python.org/packages/b3/9b/f2824009aefc5876a10d122424f83d31572ead24aaec869837a86dbd7156/RBTools-0.7.10.tar.gz"; sha256 = "7da97baa0dd447b3d83e1eed8ff41eee32d9195925ac08dc2c7b019b9b85ae8d"; };
+      doCheck = commonDoCheck;
+      buildInputs = commonBuildInputs;
+      propagatedBuildInputs = [
+      self."six"
+      self."tqdm"
+    ];
```

Comment:
i guess we would have to fix this here https://github.com/garbas/pypi2nix/blob/master/src/pypi2nix/stage3.py#L179